.github/workflows/continuous-integration.yml

```
@@ -25,7 +25,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9, 3.10, 3.11]
```

I _think_ this will need quotes.
```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11"]
```